sentiment analysis for F message

Abusayid693 opened the pull request:
sends 'f' if the message is negative

Abusayid693 committed:
Update main.py

Abusayid693 committed:
Update main.py

Abusayid693 committed:
Update main.py

Abusayid693 commented:
Whenever user sends a negative message (eg : kill everyone) , the bot replies with "f" 
![SmartSelect_20210617-024519_Discord](https://user-images.githubusercontent.com/35341198/122294837-1d231280-cf16-11eb-8df4-2aa1d2aba2e6.jpg)

Locally tested

maralihart commented:
What about a positive message or a neutral message?
How negative does it have to be?

#15

Abusayid693 commented:
Positive and neutral messages are skipped (if u say I may add some functionality for +ve and neutral) we take max probability out of all three sentiments , the sentence has to be Declarative that have a negative meaning. Apart from this there can be emotion analysis like sad messages, bored messages, angry etc, the bot may react to those messages.

Abusayid693 committed:
sentiment analysis

Reacts to positive and neutral messages with custom emoji and send a message is deemed negative enough, sends F in the chat

Abusayid693 commented:
Did few more changes please check

1. Reacts to positive and neutral messages with emojis  from dictionary ( positive and neutral ) randomly selected ,more custom emojis can be added in the dictionaries.
2. Send "F" if message is negative and having threshold above 0.5.

Locally tested 
#15

avinashupadhya99 commented on main.py:
```diff
@@ -3,12 +3,17 @@
 token = os.environ['TOKEN']
 bot = discord.Client()
 
+API_KEY="N5U7mnmq9gGLwxVjhPjOncSEJScUSG2eIh6l63XCRWc"
```
oof, probably need to add this as an environment variable

Abusayid693 committed:
Update main.py
